ingestors/tabular/ods.py

```diff
@@ -63,11 +63,22 @@ def ingest(self, file_path, entity):
             table = self.manager.make_entity("Table", parent=entity)
             table.make_id(entity.id, name)
             table.set("title", name)
+            # add workbook metadata to individual tables
+            for metadatum in [
+                "authoredAt",
+                "author",
+                "summary",
+                "generator",
+                "date",
+                "processingAgent",
+            ]:
+                table.set(metadatum, entity.get(metadatum))
             # Emit a partial table fragment with parent reference and name
             # early, so that we don't have orphan fragments in case of an error
             # in the middle of processing.
             # See https://github.com/alephdata/ingest-file/issues/171
             self.manager.emit_entity(table, fragment="initial")
+            log.debug("Sheet: %s", name)
```

Comment:
Is this intentionally left here? I think in production this would cause a lot of unnecessary logging.